apps/wallet-mobile/src/metrics/metricsManager.tsx

```diff
@@ -31,18 +31,57 @@ const infoPlugin: EnrichmentPlugin = {
 }
 
 export const makeMetricsStorage = (yoroiStorage: YoroiStorage = storage) => {
-  const enabledKey = 'metrics-enabled'
-  const consentRequestedKey = 'metrics-consentRequested'
+  const consentRequestedKey = 'metrics'
   const settingsStorage = yoroiStorage.join('appSettings/')
 
-  const enabled = {
-    read: () => settingsStorage.getItem(enabledKey).then((value) => parseBoolean(value) ?? __DEV__),
-    write: (enable: boolean) => settingsStorage.setItem(enabledKey, JSON.stringify(enable)),
-  } as const
+  const defaultConsentRequested = false
+  const defaultEnabled = __DEV__
+
+  type MetricsStorageValue = {
+    enabled: boolean
+    consentRequested: boolean
+    version: number
+    dateConsentRequested: string
```

Comment:
if we use to display now() > version (transform version to the date of the consent), I think we can drop consentRequested / consentRequested. and keep only the version. as long as we always update version whether the user skips / accepts